### DISCUSSION TITLE
Add battery voltage monitoring

### DISCUSSION THREAD

**Discussion by tedlanghorst** [tedlanghorst/OpenOBS#16] [Ideas]
We may be having problems with corrupted data or failed SD card writes due to low battery voltage at the end of deployments. Adding a circuit like below would let us monitor the battery voltage and shut down the sensor before doing any damage to the data.

https://jeelabs.org/2013/05/16/measuring-the-battery-without-draining-it

**Comment by anardek** [comment]
I think this is super smart. It also looks pretty simple to implement.

**Comment by tedlanghorst** [comment]
Now included on the new battery PCB design. Rev2 battery boards are compatible with the original sensor boards.
